qa/tasks/cephfs/filesystem.py

```diff
@@ -1530,3 +1531,32 @@ def shrink(self, new_max_mds, status=None):
         assert(new_max_mds < oldmax)
         self.set_max_mds(new_max_mds)
         return self.wait_for_daemons()
+
+    def run_scrub(self, cmd, rank=0):
+        return self.rank_tell(["scrub"] + cmd, rank)
+
+    def get_scrub_status(self, rank=0):
+        return self.run_scrub_command(["status"], rank)
+
+    def wait_until_scrub_complete(self, result=None, tag=None, rank=0, sleep=30, timeout=300):
+        # time out after "timeout" seconds and assume as done
+        if result is None:
+            result = "no active scrubs running"
```

Comment:
The result is  `None` for most of the calls made in this PR, except `_check_scrub_status`? Is that correct?

Reply:
Sorry for missing this one.
Yeah that's correct, for most case it will check the scrub status with `"no active scrubs running"` and only in few cases it won't.